msteams-platform/toolkit/debug-background-process.md

```diff
@@ -40,6 +40,8 @@ For more information, see [Node.js version compatibility table for project type]
 * Teams Toolkit prompts you to sign in to Microsoft 365 account, if you haven't signed in with your valid credentials.
 * Custom app uploading or sideloading for your developer tenant is turned on, to prevent local debug termination.
 * Teams Toolkit installs Ngrok npm package `ngrok@4.2.2` in `~/.fx/bin/ngrok`, if Ngrok isn't installed or the version doesn't match the requirement. Ngrok binary version 2.3 is applicable for bot and message extension. The Ngrok binary is managed by Ngrok npm package in `/.fx/bin/ngrok/node modules/ngrok/bin`.
+  > [!WARNING]
+  > Teams Toolkit downloads an npm package ngrok@4.3.3, which contains [ngrok](https://ngrok.com/) v2.3.40. The user must have a valid license to use ngrok. Microsoft doesn't license the use of the ngrok.
```

Comment:
Does this need to be a WARNING?

The string message needs adjusted too.

Reply:
I can help with phrasing if we don't already have another person providing the exact string.

Reply:
Hi @therealjohn ,

Can you please elaborate on the request. Are you referring to the Warning message as the string message?

i.e., Teams Toolkit downloads an npm package ngrok@4.3.3, which contains [ngrok](https://ngrok.com/) v2.3.40. The user must have a valid license to use ngrok. Microsoft doesn't license the use of the ngrok.

Thanks,
Yash

Reply:
Hi @therealjohn,

A gentle reminder.

Thanks,
Yash

Reply:
@v-ypalikila Here's my suggestion for the change: 

 > [!NOTE]
 > Teams Toolkit project templates use the ngrok@4.3.3 NPM package, which contains ngrok v2.3.40. Visit the [ngrok](https://ngrok.com/pricing) page to get a valid license.

Reply:
Updated.